app/sentry.ts

```diff
@@ -10,8 +11,10 @@ if (!config.sentry.dsn.includes('dummy')) {
             new Dedupe(),
             new CaptureConsole({
                 levels: ['error']
-            })
+            }),
+            new ApmIntegrations.Tracing(),
         ],
+        tracesSampleRate: config.sentry.tracesSampleRate, 
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5948471)